Containerfile.postgres

```diff
@@ -0,0 +1,5 @@
+# Use offical quay.io postgres image
+FROM quay.io/sclorg/postgresql-13-c9s:latest
```

Comment:
I was anticipating that you'd base our containerfile on the one that builds this scl image, not just use the image directly, because this leaves us at the mercy of changes that can happen to this image w/o our control.